lib/components/molecules/AtomicCarousel/AtomicCarousel.story.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, number, text } from '@storybook/addon-knobs';
+// import { action } from '@storybook/addon-actions';
+
+import defaultConfig from './AtomicCarousel.mock';
+import Image from '../../atoms/Image';
+import image1 from './images/1.jpg';
+import image2 from './images/2.jpg';
+import image3 from './images/3.jpg';
+import image4 from './images/4.jpg';
+
+// Import Styled Component to showcase variations
+import AtomicCarousel, { AtomicCarouselVanilla } from '.';
+
+storiesOf('Molecules/AtomicCarousel', module)
+  .addParameters({ jest: ['AtomicCarousel', 'AtomicCarouselVanilla'] })
+  .add('AtomicCarouselVanilla Knobs', () => (
+    <AtomicCarouselVanilla
+      {...defaultConfig}
+      dots={boolean('Dots', true)}
```

Comment:
There are too many props on the main component, some of it can be delegated to compound component